### PR TITLE
Remove redundant method from class GenericVector

### PR DESCRIPTION
**Pull request by stweil**
length() is not needed: it can be replaced by size().

Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by zdenop** [comment]
thanks!

File: src/ccmain/control.cpp [hunk 1]
```diff
@@ -721,9 +721,9 @@ void Tesseract::blamer_pass(PAGE_RES* page_res) {
         static_cast<IncorrectResultReason>(bl)),
         page_res->blame_reasons[bl]);
   }
-  if (page_res->misadaption_log.length() > 0) {
+  if (page_res->misadaption_log.size() > 0) {
```
Comment (amitdo) [inline src/ccmain/control.cpp]: Maybe you can drop the `> 0` part.

Comment (egorpugin) [reply src/ccmain/control.cpp]: `.empty()` is better.

Comment (stweil) [reply src/ccmain/control.cpp]: Yes. In this case `! .empy()`, of course. `git grep "length().*0"` and `git grep "size().*0"` shows more code locations which could use `empty`.

**Pull request merged**